### ISSUE TITLE
FCA unit tests in incorrect package

### ISSUE THREAD

**Issue by mheuer-nmdp** [nmdp-bioinformatics/ngs#125]
Some FCA unit tests are in an incorrect package, i.e.

```
fca/src/test/java/org/nmdp/fca/ConceptLatticeTest.java
fca/src/test/java/org/nmdp/fca/ConceptTest.java
fca/src/test/java/org/nmdp/fca/ConnectedGraphTest.java
fca/src/test/java/org/nmdp/fca/PrunerTest.java
fca/src/test/java/org/nmdp/fca/VertexTest.java
```
should be refactored to package ```org.nmdp.ngs.fca```.

**Comment by mheuer-nmdp** [comment]
If it helps, I can issue a new pull request for this after #136 is merged

**Comment by ckennedy-nmdp** [comment]
Yes, please. Merging now.
